.npmignore

```diff
@@ -0,0 +1 @@
+tools
```

Comment:
whether `docs` dir needs to be added to this file?

Reply:
Sure, I've added it too. I noticed `tools` in linting because of this rule: https://github.com/mysticatea/eslint-plugin-node/blob/v8.0.1/docs/rules/no-unpublished-require.md . But I've added `docs` now too.